pset-iterators.js

```diff
@@ -184,6 +234,15 @@ console.log('Problem 7:')
 
 // Add your code below this line
 
+const shortTermPresidents = presidents.filter(president => {
+	if(president.left_office === null){
+		president.left_office = 2020
+}
```

Comment:
Way to take edge cases into consideration! Hopefully your prediction is right :)